PVList.md

```diff
@@ -0,0 +1,354 @@
+# IPMI Manager Process Variables naming list
+
+This document have the purpose of making easier to understand the substitution
+tree on this IOC and to check if the [LNLS's naming convention](https://wiki-sirius.lnls.br/mediawiki/index.php/Machine:Naming_System)
+of is being followed. It is also useful for any user to check what PVs are
+available.
+Each new record created or aliased from the ipmiComm module must be documented
+here.
```

Comment:
```suggestion
This document has the purpose of making it easier to understand the PV
substitution tree in this IOC and to follow LNLS's naming convention. It is
also useful for any user to check which PVs are available.

Each new record created, or aliased from the ipmiComm module, must be
documented here.
```

I'd remove the link to wiki Sirius because it is not available in the Internet.

Hint: when writing this in your (neo)vim, select the text with `Ctrl+v` and format it with `gq`. This will hard-wrap the text for you following the maximum column size. With a good language server support, you should be able to do this with code as well.

Reply:
> I'd remove the link to wiki Sirius because it is not available in the Internet.
Nice catch. I'll also remove it from the commit message, thanks!